xacro/__init__.py

```diff
@@ -298,7 +296,7 @@ def next(self):
 
 
 all_includes = []
-include_no_matches_msg = """Include tag's filename spec \"{}\" matched no files."""
+include_no_matches_msg = 'Include tag\'s filename spec "{}" matched no files.'
```

Comment:
The only reason to change most of the quotes is that one of the default ament python linters insists on single-quotes for most everything.  If you don't think that is worthwhile, then we won't be able to enable the linters.  If that's the case, that's fine, we can just drop most of this PR (though I still think it would be nice to remove the Python 2 fallback code).  Let us know what you want to do here.

Reply:
I already merged the uncritical parts of this PR, particularly the removal of the Python 2 fallback code. Regarding quotes, I'm hesitating to change all the code and risk new issues. If the ament linter is too restrictive, we shouldn't use it. Apart from the quotes there are some other pep8 issues (too long lines, because the xacro code is formated for 127 chars line width, while the ament linter checks for 99 chars only), which I don't want to resolve.

Reply:
All right, that's totally fine with me.  In that case, I think we can close this out.